.dockerignore

```diff
@@ -0,0 +1,5 @@
+.git
+.gitignore
+.next
+.npmignore
+node_modules
```

Comment:
You may want to also add `dist/` here as you are going to be building the content of this fresh when running `yarn build`